src/azul/chalice.py

```diff
@@ -617,3 +618,25 @@ def lambda_context(self) -> LambdaContext:
     @property
     def current_request(self) -> AzulRequest:
         return self.app.current_request
+
+    def server_side_sleep(self, max_seconds: int | float) -> float:
```

Comment:
```suggestion
    def server_side_sleep(self, max_seconds: float) -> float:
```

Time in Python is always a float. Supporting `int` may seem like a convenience but just leads to sloppy use of this function.